src/Network/Network/help/Set-AzApplicationGatewaySslCertificate.md

```diff
@@ -27,7 +27,8 @@ The **Set-AzApplicationGatewaySslCertificate** cmdlet updates an SSL certificate
 ### Example 1: Update an existing SSL certificate on Application Gateway
 ```
 PS C:\> $appGW = Get-AzApplicationGateway -Name "ApplicationGateway01" -ResourceGroupName "ResourceGroup01"
-PS C:\> $password = ConvertTo-SecureString "P@ssw0rd" -AsPlainText -Force
+PS C:\> $passwordString = "P@ssw0rd"
```

Comment:
what I mean is not to list password content in the doc, but use a variable as the replacement such as

PS C:\> $passwordPlainString
PS C:\> $password = ConvertTo-SecureString $passwordPlainString -AsPlainText -Force

Reply:
And no need to declare the `$passwordPlainString` variable as well right?